Dockerfile

```diff
@@ -87,6 +87,9 @@ RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 RUN apt-get install -y nodejs
 RUN npm install -g svgo
 
+# Installing bids-validator
+RUN npm isntall -g bids-validator@1.1.0
```

Comment:
```suggestion
RUN npm install -g bids-validator@1.1.0
```

Reply:
Looks like `npm` is forgiving of misspellings?

https://circleci.com/gh/chrisfilo/fmriprep/782